assets/app/view/game/merge.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+require 'view/game/undo_and_pass'
+
+module View
+  module Game
+    class Merge < Snabberb::Component
+      include Actionable
+
+      def render
+        corporation = @game.mergable_candidates.first
+        merge = lambda do
+          process_action(Engine::Action::Merge.new(
+            corporation.player ? @game.current_entity : @game.ndm,
+            corporation: corporation,
+          ))
+        end
+
+        children = [h(:div, "Merge candidates: #{@game.mergable_candidates.map(&:name)}")]
+        children << render_description(corporation)
+        children << h(:div, [h(:button, { on: { click: merge } }, 'Merge into NdM')])
+        children << h(Corporation, corporation: corporation)
+        children << h(Corporation, corporation: @game.ndm)
+        h(:div, children)
+      end
+
+      def render_description(corporation)
+        player = corporation.player
+        player ||= @game.ndm.player
```

Comment:
Bit confused on this, should probably be a needs into the view to say the player name.

Reply:
See pictures below. The idea here is the merge candidate is shown, and the president of it makes the decision if it should merge. In case it has no president, it is a non-started one, and then NdM should make the decision. (Note my remark below that this is very likely automatic.)

I know realize that this is probably an error - only if the corporation is floated the player can decide. Non-floated is automatically merged if one if players decline. If non-floated and non-sold presidency, that is chosen if players decline. And if there is two non-floated eligible (both non-sold, or sold) - then NdM gets to choose.

Reply:
As can be seen from the screen shots, the merge candidate is shown underlined in the Order, as well as in text above the button. Only time it is not shown underlined is if it is a non-floated corporation that is up for merge.

Reply:
Improved the code a bit to make it clearer.